src/MOI_wrapper.jl

```diff
@@ -22,7 +22,10 @@ struct CbcModelFormat
     obj::Vector{Float64}
     row_lb::Vector{Float64}
     row_ub::Vector{Float64}
-    function CbcModelFormat(num_rows::Int, num_cols::Int)
+    # `num_cols` is `Int64` if the model is copied from a model create with
```

Comment:
I had to read this comment multiple times to understand it. I think it would be clearer to add a note above the call to `new` that an `InexactError` might occur if `num_rows` or `num_cols` is too large.